scripts/ci/run_e2e_roundtrip_tests.py

```diff
@@ -19,32 +20,60 @@
 
 ARCHETYPES_PATH = "crates/re_types/definitions/rerun/archetypes"
 
+# TODO(andreas): Remove these
+cpp_opt_out = ["points2d", "transform3d"]
+
 
 def main() -> None:
     parser = argparse.ArgumentParser(description="Run our end-to-end cross-language roundtrip tests for all SDK")
-    parser.add_argument("--no-build", action="store_true", help="Skip building rerun-sdk")
+    parser.add_argument("--no-py-build", action="store_true", help="Skip building rerun-sdk")
     parser.add_argument("--full-dump", action="store_true", help="Dump both rrd files as tables")
-    parser.add_argument("--release", action="store_true", help="Run cargo invocations with --release")
+    parser.add_argument(
+        "--release",
+        action="store_true",
+        help="Run cargo invocations with --release and CMake with `-DCMAKE_BUILD_TYPE=Release` & `--config Release`",
+    )
     parser.add_argument("--target", type=str, default=None, help="Target used for cargo invocations")
     parser.add_argument("--target-dir", type=str, default=None, help="Target directory used for cargo invocations")
 
     args = parser.parse_args()
 
-    if args.no_build:
-        print("Skipping building rerun-sdk - assuming it is already built and up-to-date!")
-    else:
-        build_env = os.environ.copy()
-        if "RUST_LOG" in build_env:
-            del build_env["RUST_LOG"]  # The user likely only meant it for the actual tests; not the setup
+    build_env = os.environ.copy()
+    if "RUST_LOG" in build_env:
+        del build_env["RUST_LOG"]  # The user likely only meant it for the actual tests; not the setup
 
+    if args.no_py_build:
+        print("Skipping building python rerun-sdk - assuming it is already built and up-to-date!")
+    else:
         print("----------------------------------------------------------")
-        print("Building rerun-sdk…")
+        print("Building rerun-sdk for Python…")
         start_time = time.time()
-        subprocess.Popen(["just", "py-build"], env=build_env).wait()
+        returncode = subprocess.Popen(["just", "py-build"], env=build_env).wait()
+        assert returncode == 0, f"Python rerun-sdk build failed with exit code {returncode}"
         elapsed = time.time() - start_time
-        print(f"rerun-sdk built in {elapsed:.1f} seconds")
+        print(f"rerun-sdk for Python built in {elapsed:.1f} seconds")
         print("")
 
+    print("----------------------------------------------------------")
```

Comment:
Seems like an `args.no_cpp_build` would make sense.

Reply:
you're right, let's add it now and not when someone needs it urgently! :)